tests/integration/conftest.py

```diff
@@ -139,9 +141,15 @@ def __call__(self) -> ContextManager[ProtostarFixture]:
 
 @pytest.fixture(name="create_protostar_project", scope="module")
 def create_protostar_project_fixture(
+    request: FixtureRequest,
     session_mocker: MockerFixture,
     tmp_path_factory: TempPathFactory,
 ) -> CreateProtostarProjectFixture:
+    cairo_version = (
+        CairoVersion.cairo0 if not hasattr(request, "param") else request.param
```

Comment:
Can you check if there's a way to do this without using `hasattr`? I think we want to avoid it if possible.

Reply:
I could check the class instance but I think that would make it worse. From source code of pytest:
```python
# Notes on the type of `param`:
# -`request.param` is only defined in parametrized fixtures, and will raise
#   AttributeError otherwise. Python typing has no notion of "undefined", so
#   this cannot be reflected in the type.
# - Technically `param` is only (possibly) defined on SubRequest, not
#   FixtureRequest, but the typing of that is still in flux so this cheats.
# - In the future we might consider using a generic for the param type, but
#   for now just using Any.
self.param: Any
```